lib/runner/extensions/event.command.js

```diff
@@ -352,9 +352,11 @@ module.exports = {
                     result && result.response && (result.response = new sdk.Response(result.response));
 
                     // persist the pm.variables for the next script
-                    payload.context._variables = new sdk.VariableScope(result._variables);
+                    result && result._variables &&
+                        (payload.context._variables = new sdk.VariableScope(result._variables));
```

Comment:
The type safety should be handled within the Collection SDK.

Reply:
Not future nitpick. I think that’s what should be done now. 

new VariableScope should just work without arguments. There was a test for this IMHO. 

Is the “result” object missing or is the bug in variable scope constructor?